src/OpenApi/Tests/fixtures/async-generation/expected/ClientAsync.php

```diff
@@ -42,7 +42,7 @@ public function getTestById(int $id, string $fetch = self::FETCH_OBJECT)
     /**
      * @param string $fetch Fetch mode to use (can be OBJECT or RESPONSE)
      *
-     * @return \Amp\Promise<null|\Jane\OpenApi\Tests\Expected\Model\Schema[]|\Amp\Artax\Response>
+     * @return \Amp\Promise<\Jane\OpenApi\Tests\Expected\Model\Schema[]|\Amp\Artax\Response|null>
```

Comment:
This is the one that triggers in CI, don't understand only this one, and only for PHP != 7.3